rust/src/lib.rs

```diff
@@ -1012,19 +1012,19 @@ pub unsafe extern "C" fn zcashlc_get_sapling_receiver_for_unified_address(
 /// - `address` must be non-null and must point to a null-terminated UTF-8 string.
 /// - The memory referenced by `address` must not be mutated for the duration of the function call.
 #[no_mangle]
-pub unsafe extern "C" fn zcashlc_is_valid_shielded_address(
+pub unsafe extern "C" fn zcashlc_is_valid_sapling_address(
```

Comment:
Why do we have this for Sapling and not for Orchard? What else is to do for Orchard?

Reply:
This is for validating a Sapling address encoding. We don't have an Orchard address encoding; we use UAs for that, which is what `zcashlc_is_valid_unified_address` is for.